manifests/public.yaml

```diff
@@ -89,6 +89,7 @@ spec:
   - name: salt-master-config
     image: sles12/velum:__TAG__
     command: ["sh", "-c", "umask 377;
+                           rmdir /salt-master-credentials/55-returner-credentials.conf;
```

Comment:
Why this is directory and not file?

Reply:
The very first time k8s loads this manifest, this file does not exist on the host path, and is created as a directory. The init container removes this directory and writes the file. It's far from ideal, but does the trick.

Reply:
wtf! I think you should add a comment explaining this...